### DISCUSSION TITLE
Swiper does not work for new code

### DISCUSSION THREAD

**Discussion by skater2014** [nolimits4web/swiper#5262] [General]
reading css for swiper is good

but not reading javascripts for swiper 

I have tried CDN also in header area 

i am not sure what to do

**Comment by Arhell** [comment]
Hi.
Here are some use cases
https://swiperjs.com/get-started#installation

If it doesn't work, just download the source and connect it to the project so as not to use cdn

**Reply by skater2014** [reply]
ok i try new version for swiper ?

**Reply by skater2014** [reply]
i already downloaded it does not read js 
but css can read

**Reply by skater2014** [reply]
https://kaztokyo.sakura.ne.jp/gallary/

here is problem not reading js but read css for swiper slider so far.
Alreadt downloaded css and js for swiper slider 
I also tried to read CDN for reading js but I think it can not read

**Reply by skater2014** [reply]
here is my scripts all written may cause problem?




jQuery(document).ready(function($){

	$('.head-nav .menu').slicknav({label: '' , prependTo: 'header#head .container'});

	$('.head-nav ul li').hover(function(){
		$('.sub-menu:first, .children:first',this).stop(true,true).slideDown('fast');
	},
	function(){
		$('.sub-menu:first, .children:first',this).stop(true,true).slideUp('fast');
	});

	$('.page-wrap').css('margin-top',$('header#head').outerHeight());

	$(window).bind('scroll', function () {
	    if ($(window).scrollTop() > 150) {
	        $('#head').addClass('fixed')
	    } else  {
	        $('#head').removeClass('fixed');
	    }
	});

//home slick slider for blog page

	get_blogbox_height();

	$('#home-slider').slick();

	var grid = jQuery('.home-posts').imagesLoaded( function() {
		get_blogbox_height();
		grid.isotope({
		  // options
		  itemSelector: '.archive-post-box',
		  layoutMode: 'fitRows',
		  filter: '*',
		});
	});

});

//home slick slider for home page

function get_blogbox_height(){
	/*var w = jQuery('.archive-post-feature').width() * .5625;
	jQuery('.archive-post-feature').height(w);*/

	var t_w = jQuery('.archive-post-feature').width();
	var w = (t_w == 100 ? $(window).width() : t_w) * .5625;
	jQuery('.archive-post-feature').height(w);

	//var w2 = jQuery('.blog-feature').width() * .559;
	//jQuery('.blog-feature').height(w2);

	//jQuery(".blog-excerpt p").dotdotdot({
		//height: w2+20
	//});

	var w3 = jQuery('.home-slide-feature').height() - 100;
	if (jQuery(window).width() > 767) {
		jQuery(".home-slide-text").dotdotdot({
			height: w3
		});
	}
	

	// jQuery('.post_content iframe').height(jQuery('.post_content iframe').width() * .5625);

}

jQuery(window).resize(function(){
	get_blogbox_height();
})

jQuery(window).load(function() {
	//jQuery('.page-wrap').css('margin-top',jQuery('header#head').outerHeight());

	jQuery('.home-slider').removeClass('invisibility');

	jQuery('.home-cats-selection ul li a').on('click',function(e){
		e.preventDefault();
		jQuery('.home-cats-selection ul li a').removeClass('active');
		jQuery(this).addClass('active');

		var target = jQuery(this).attr('href');
		jQuery('.home-posts').isotope({ filter: target })
	})
});



//Initialize Swiper for slider for gallary page

const swiper = new Swiper('.swiper', {
  // Optional parameters
  direction: 'horizontal',
  slidesPerView: "auto",
  loop: true,
  

  // If we need pagination
  pagination: {
    el: '.swiper-pagination',
    type: 'bullets', //ページネーションの種類
    clickable: true, //クリックに反応させる
  },

  // Navigation arrows
  navigation: {
    nextEl: '.swiper-button-next',
    prevEl: '.swiper-button-prev',
  },

  // And if we need scrollbar
  scrollbar: {
    el: '.swiper-scrollbar',
  },
});


//News ticker for header.php
jQuery(document).ready(function($) {
      $('.gallary-slider').slick({
    arrows: false,
    autoplay:true, //全部小文字
    autoplaySpeed: 2000,　//全部小文字
    vertical: true,
  });  
});

**Reply by skater2014** [reply]
https://codepen.io/kazusano99/pen/dyVYPVJ

this one work but not wordpress for swiper slider
